pyslackersweb/website/__init__.py

```diff
@@ -3,32 +3,23 @@
 
 from aiohttp import web
 from aiohttp_jinja2 import setup as jinja2_setup, request_processor
-from aiohttp_remotes import XForwardedRelaxed, ForwardedRelaxed
 from jinja2 import FileSystemLoader
 from jinja2.filters import FILTERS
 
-from .contexts import background_jobs, client_session, apscheduler, slack_client
+from pyslackersweb import settings
```

Comment:
`from . import settings`

Reply:
Settings is a package up, are you wanting to use relative imports? `from .. import settings`?

Reply:
Ah my bad. Then I don't really care if we use relative or absolute